server/StudySharp.Domain/Models/AppearanceType.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace StudySharp.Domain.Models
+{
+    public enum AppearanceType
```

Comment:
Are you sure we won't have access modifiers for other entities in the future?
Maybe add "Course" to this name, or change it to more corresponding to courses